tests/Agent/IntegrationTests/UnboundedIntegrationTests/NServiceBus/NServiceBusThrowingCommandHandlerTests.cs

```diff
@@ -0,0 +1,179 @@
+    //{
+    //    public NServiceBusTestsFW462(ConsoleDynamicMethodFixtureFW462 fixture, ITestOutputHelper output)
+    //        : base(fixture, output)
+    //    {
+    //    }
+    //}
+
+    [NetFrameworkTest]
+    public class NServiceBusThrowingCommandHandlerTestsFW471 : NServiceBusThrowingCommandHandlerTestsBase<ConsoleDynamicMethodFixtureFW471>
+    {
+        public NServiceBusThrowingCommandHandlerTestsFW471(ConsoleDynamicMethodFixtureFW471 fixture, ITestOutputHelper output)
+            : base(fixture, output)
+        {
+        }
+    }
+
+    [NetFrameworkTest]
+    public class NServiceBusThrowingCommandHandlerTestsFW48 : NServiceBusThrowingCommandHandlerTestsBase<ConsoleDynamicMethodFixtureFW48>
+    {
+        public NServiceBusThrowingCommandHandlerTestsFW48(ConsoleDynamicMethodFixtureFW48 fixture, ITestOutputHelper output)
+            : base(fixture, output)
+        {
+        }
+    }
+
+    [NetCoreTest]
+    public class NServiceBusThrowingCommandHandlerTestsCore21 : NServiceBusThrowingCommandHandlerTestsBase<ConsoleDynamicMethodFixtureCore21>
+    {
+        public NServiceBusThrowingCommandHandlerTestsCore21(ConsoleDynamicMethodFixtureCore21 fixture, ITestOutputHelper output)
+            : base(fixture, output)
+        {
+        }
+    }
+
+    [NetCoreTest]
+    public class NServiceBusThrowingCommandHandlerTestsCore22 : NServiceBusThrowingCommandHandlerTestsBase<ConsoleDynamicMethodFixtureCore22>
+    {
+        public NServiceBusThrowingCommandHandlerTestsCore22(ConsoleDynamicMethodFixtureCore22 fixture, ITestOutputHelper output)
+            : base(fixture, output)
+        {
+        }
+    }
+
+    [NetCoreTest]
+    public class NServiceBusThrowingCommandHandlerTestsCore31 : NServiceBusThrowingCommandHandlerTestsBase<ConsoleDynamicMethodFixtureCore31>
+    {
+        public NServiceBusThrowingCommandHandlerTestsCore31(ConsoleDynamicMethodFixtureCore31 fixture, ITestOutputHelper output)
+            : base(fixture, output)
+        {
+        }
+    }
+
+    [NetCoreTest]
+    public class NServiceBusThrowingCommandHandlerTestsCore50 : NServiceBusThrowingCommandHandlerTestsBase<ConsoleDynamicMethodFixtureCore50>
+    {
+        public NServiceBusThrowingCommandHandlerTestsCore50(ConsoleDynamicMethodFixtureCore50 fixture, ITestOutputHelper output)
+            : base(fixture, output)
+        {
+        }
+    }
+
+    [NetCoreTest]
+    public class NServiceBusThrowingCommandHandlerTestsCore60 : NServiceBusThrowingCommandHandlerTestsBase<ConsoleDynamicMethodFixtureCore60>
+    {
+        public NServiceBusThrowingCommandHandlerTestsCore60(ConsoleDynamicMethodFixtureCore60 fixture, ITestOutputHelper output)
+            : base(fixture, output)
+        {
+        }
+    }
+
+    [NetCoreTest]
+    public class NServiceBusThrowingCommandHandlerTestsCoreLatest : NServiceBusThrowingCommandHandlerTestsBase<ConsoleDynamicMethodFixtureCoreLatest>
```

Comment:
My thinking by targeting .NET Core `latest` and `6.0` is that it somewhat future proofs this when the fixtures get updated?